TheCircleBackend/TheCircleBackend/Hubs/ViewerHub.cs

```diff
@@ -127,7 +127,7 @@ private async Task AllowUserNotToWatch(bool isAllowed, string ConnectId, int str
 
         private bool CheckMaxViews(int watcherId)
         {
-            return viewerRepository.GetCurrentViewerCount(watcherId) < 4;
+            return viewerRepository.GetCurrentViewerCount(watcherId) <= 4;
```

Comment:
Vlak na de methode wordt er nog een count toegevoegd. 
In deze versie, wordt de viewcount van de gebruiker naar 5 gepushed.
In de oude versie wordt dat niet gedaan en stopt het bij 4.

Reply:
Oude proces:
bij 3 < 4
3 + 1 = 4, de nieuwe versie
bij 4 < 4
 Viewer niet toegevoegd.

Nieuwe versie
4 <= 4 = true
4 + 1 = 5, de nieuwe viewcount